clang/bindings/python/tests/cindex/test_cursor.py

```diff
@@ -570,6 +570,26 @@ def test_enum_values_cpp(self):
         self.assertEqual(ham.kind, CursorKind.ENUM_CONSTANT_DECL)
         self.assertEqual(ham.enum_value, 0x10000000000)
 
+    def test_enum_values_on_elaborated_type(self):
+        tu = get_tu(
+            "using myUType = unsigned char; enum TEST : myUType { SPAM = 1, HAM = 0xff; }",
```

Comment:
The semicolon goes outside the curly braces ;)
```suggestion
            "using myUType = unsigned char; enum TEST : myUType { SPAM = 1, HAM = 0xff };",
```